src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -28,10 +29,21 @@ public function __construct(AuthorizationCheckerInterface $securityChecker = nul
         $this->securityChecker = $securityChecker;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('is_granted', array($this, 'isGranted')),
+            new \Twig_SimpleFunction('user', array($this, 'getUser')),
+        );
+    }
+
     public function isGranted($role, $object = null, $field = null)
     {
         if (null === $this->securityChecker) {
-            return false;
+            throw new \RuntimeException('The "is_granted" function is not available.');
```

Comment:
I think it's more a LogicException. This exception does rely on a misconfiguration of the code, so it could be found on compile time rather than runtime.